### ISSUE TITLE
Cant use the standard banner

### ISSUE THREAD

**Issue by Enkeimaru** [WarEmu/WarBugs#16032]
**Expected behavior and actual behavior:**
Expected behavior:
Activating the Standard Banner

Actual behavior:
The banner would spawn.

**Steps to reproduce the problem:**

step by step:

Guild rank 16
assign battle standard to member
equip battle standard
put in hotbar
use
dont work

**Testing Screenshots/Videos/Evidences (always needed):**

pictures down here..
https://imgur.com/NymIzUA
https://imgur.com/WhrY50j
https://imgur.com/oPohJAO
https://imgur.com/XjCuvYU
https://imgur.com/vPaxwPf

**Comment by Enkeimaru** [comment]
How long does it take till there comes a response?

**Comment by ullagor** [comment]
Same issue. I really hope the staff considers this as a disadvantage for newly formed guilds.

**Comment by Enkeimaru** [comment]
Yup i hope so.. this is a gamechanging bug.. guilds without the banners got a pretty big disadvantage.. i hope they make work out of it.

**Comment by Rivenspeak** [comment]
I have the same issue.  See attached screenshots.  Guild lvl 13 with all three tactic slots filled.  Correct banner type, equipped, and on action bar.
![Drystelle_004](https://user-images.githubusercontent.com/45123644/82946658-bdee0b00-9f6c-11ea-9129-e17e2e05a705.jpg)

**Comment by Rivenspeak** [comment]
Update to note that after guild rank 18, problem still applies now to Warped and Bloodied standards.
